### DISCUSSION TITLE
I do not know how to open the gdb debugger config.

### DISCUSSION THREAD

**Discussion by ishax-kos** [intellij-dlanguage/intellij-dlanguage#622] [Q&A]
I'm not used to intellij. How do I return to the window where you input a path to the gdb debugger? I couldn't find anything through google. It could be that I dont know what to search for.

**Comment by SingingBush** [comment]
it's in: _Settings_ > _Languages & Frameworks_ > _D Tools_ > _GDB_

![image](https://user-images.githubusercontent.com/870567/118546432-db088480-b74f-11eb-8b21-c75e9186de6b.png)
